python_modules/dagster/dagster/core/asset_defs/assets_from_modules.py

```diff
@@ -71,7 +71,10 @@ def assets_and_source_assets_from_modules(
 
 
 def assets_from_modules(
-    modules: Iterable[ModuleType], extra_source_assets: Optional[Sequence[SourceAsset]] = None
+    modules: Iterable[ModuleType],
+    group_name: Optional[str] = None,
+    asset_key_prefix: Optional[str] = None,
```

Comment:
Thoughts on just calling this `key_prefix` for brevity? Also, these need documentation.

Reply:
Changed this to `key_prefix` and added docstrings!